exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/DrillOperatorTable.java

```diff
@@ -157,9 +159,23 @@ private void populateWrappedCalciteOperators() {
       } else if(calciteOperator instanceof SqlFunction) {
         wrapper = new DrillCalciteSqlFunctionWrapper((SqlFunction) calciteOperator,
             getFunctionListWithInference(calciteOperator.getName()));
+      } else if(calciteOperator instanceof SqlBetweenOperator) {
+        // During the procedure of converting to RexNode,
+        // StandardConvertletTable.convertBetween expects the SqlOperator to be a subclass of SqlBetweenOperator
+        final SqlBetweenOperator sqlBetweenOperator = (SqlBetweenOperator) calciteOperator;
+        wrapper = new SqlBetweenOperator(sqlBetweenOperator.flag, sqlBetweenOperator.isNegated()) {
+          @Override
+          public boolean checkOperandTypes(
```

Comment:
Are you essentially saying that you want to disable checkOperandType enforced in Calcite for BETWEEN operator here? Why?